Automattic/jetpack
# AI Client: expose askQuestion. Import and use from Jetpack plugin

~# DO NOT MERGE Depends on https://github.com/Automattic/jetpack/pull/31944~

Fixes https://github.com/Automattic/jetpack/issues/31945

## Proposed changes:

* AI Client: expose askQuestion. Import and use from Jetpack plugin

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






Test the changes by using the AI Assistant block

* Go to the block editor
* Create an AI Assistant block instance
* Request something
* Confirm the block works as expected
* Take a look at the debug() lines. There is a new `ask-question` namespace to check the function functionality.

<img width="759" alt="Screenshot 2023-07-20 at 17 53 21" src="https://github.com/Automattic/jetpack/assets/77539/2cef536c-299e-416b-b9c4-39723d03f985">


* Switch to GPT4 from the playground panel.

<img width="1105" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/25dc1dda-99e3-4e0d-b4b8-8ab5d58db850">

* Confirm it sets the request properly

<img width="634" alt="Screenshot 2023-07-24 at 14 50 54" src="https://github.com/Automattic/jetpack/assets/77539/b9230d24-4c3a-49b5-9d17-2518079b6ba7">
<img width="798" alt="Screenshot 2023-07-24 at 14 50 35" src="https://github.com/Automattic/jetpack/assets/77539/be407577-a559-49f7-ab51-72cfc57c567d">
